pkg/omf/omf.fish

```diff
@@ -118,13 +122,26 @@ function omf -d "Oh My Fish"
       end
 
     case "t" "theme"
+      # Note: wasn't tested on Darwin/MacOSX
       if test (count $argv) -eq 1
```

Comment:
Of course you don't want it in the code base...
I'm trying to see if any of my Mac friends use Fish (shame on them twice if they don't) and can help me test if the changes are compatible with OSX tools. Since I don't trust that any of them will be helpful, I'm already creating an OSX VirtualBox for this check...
I'd prefer to test it properly before removing this line (originally I was hoping that one of you guys uses Mac).